.github/workflows/changelog.yml

```diff
@@ -0,0 +1,19 @@
+name: bandersnatch_changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
+
+jobs:
+  build:
+    name: Changelog Entry Check
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Grep CHANGES.md for PR number
+      if: contains(github.event.pull_request.labels.*.name, 'skip news') != true
```

Comment:
This is a great addition and fills a big void.

Reply:
I literally stole this idea from pypa/pip and python/cpython but thanks :D